test/unittests/server/service/service_test.py

```diff
@@ -95,6 +95,12 @@ def test_search(self):
         assert info == {'77777777777': {'content': '\n[options]\n    use_Qt=True\n'}
                                 }
 
+        info = self.search_service.search_packages(ref2, list_only=True)
+        self.assertEqual(info, {'12345587754': {}})
+
+        info = self.search_service.search_packages(ref3, list_only=True)
+        self.assertEqual(info, {'77777777777': {}})
```

Comment:
```suggestion
        assert info ==  {'12345587754': {}}

        info = self.search_service.search_packages(ref3, list_only=True)
        assert info == {'77777777777': {}}
```
This test no longer inherits from ``unittest.TestCase``, needs to move to pytest.